docs/src/main/asciidoc/verifier/introduction.adoc

```diff
@@ -817,3 +817,27 @@ The rest of the flow looks the same.
 
 Yes! Check out the https://cloud.spring.io/spring-cloud-contract/spring-cloud-contract.html#_different_base_classes_for_contracts[Different base classes for contracts] sections
 of either Gradle or Maven plugins.
+
+==== How can I debug the request/response being sent by the generated tests client?
+
+The generated tests all boil down to RestAssured in some form or fashion which relies on https://hc.apache.org/httpcomponents-client-ga/[Apache HttpClient].  HttpClient has a facility called https://hc.apache.org/httpcomponents-client-ga/logging.html#Wire_Logging[wire logging] which logs the entire request and response to HttpClient.  An example log4j2.xml configurtion could look like this
+[source,xml,indent=0]
+----
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I just tried it with my setup which is basically just the plugin and SCC libraries I need and that isn't picked up at all.  Which isn't all that shocking.  I do realize I'm not the intended case for SCC though so I think you're suggestion maps to the intended use case, doing both client and server sides of the equation.  I'm just using the plugin in EXPLICIT mode.